### PR TITLE
Think: add swift version for group_video_call;

### PR DESCRIPTION
**Pull request by jk-swift**

### CODE REVIEW

File: Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+2. 若您并未与专属客户经理取得联系那么请按后续步骤获取Appkey
+
+3. 首先在 [网易云信](https://id.163yun.com/register?h=media&t=media&clueFrom=nim&from=bdjjnim0035&referrer=https://app.yunxin.163.com/?clueFrom=nim&from=bdjjnim0035) 注册账号
+
+4. 然后在「应用」一栏中创建您的项目
+5. 等待专属客户经理联系您，并向他获取Appkey
+
+6. 将AppKey填写进Config.swift
+
+```objective-c
```
Comment (WenchaoD) [inline Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/README.md:34]:
```swift

File: Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/NERtcSample-GroupVideoCall-iOS-Swift/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
```
Comment (WenchaoD) [inline Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/NERtcSample-GroupVideoCall-iOS-Swift/AppDelegate.swift:18]: 发起权限

Comment (jk-swift) [reply Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/NERtcSample-GroupVideoCall-iOS-Swift/AppDelegate.swift:25]: 1

File: Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/README.md (hunk 1 above)
Comment (jk-swift) [reply Group-Video/NERtcSample-GroupVideoCall-iOS-Swift/README.md]: 1

**Pull request merged**